pretty.js

```diff
@@ -46,6 +46,13 @@ function filter (value) {
   return result
 }
 
+function isPinoLine (line) {
+  var requiredProperties = [ 'time', 'name', 'hostname', 'pid' ]
```

Comment:
time is not required anymore https://github.com/mcollina/pino/pull/91 :).
Maybe we should adjust this further.